nipype/interfaces/setup.py

```diff
@@ -24,6 +24,7 @@ def configuration(parent_package='', top_path=None):
     config.add_subpackage('mne')
     config.add_subpackage('mrtrix')
     config.add_subpackage('mrtrix3')
+    config.add_subpackage('niftyseg')
```

Comment:
Niftyreg/Niftyseg/Niftyfit are different packages that you install separately. Do you think having only one package won't be odd for the user? What do you think?

I could remove probably the base and use the same commands from niftyreg.base in this package. Or do you mean moving all the code into the same folder?